pom.xml

```diff
@@ -49,12 +56,35 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.21</version>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
```

Comment:
I'm not sure if we have to keep this dependencies for this h\w

Reply:
Done